decidim-admin/app/packs/src/decidim/admin/choose_language.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable no-invalid-this */
+/* eslint-disable require-jsdoc */
 
-$(() => {
-  $("select.language-change").change(function () {
-    let $select = $(this);
-    let targetTabPaneSelector = $select.val();
-    let $tabsContent = $select.parent().parent().siblings();
+export default function initLanguageChangeSelect(elements) {
+  elements.forEach((select) => {
+    select.addEventListener("click", () => {
```

Comment:
I actually made a mistake in my previous review as this should be a change listener, i.e.
```suggestion
    select.addEventListener("change", () => {
```

Sorry about that, should double check before posting... :disappointed:

Reply:
Yeah, I was a bit confused about this, but it turned out that "click" also works well 😁.